packages/web-components/src/components/cta-section/cta-section-item-heading.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings.js';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import DDSContentItemHeading from '../content-item/content-item-heading';
+import styles from './cta-section.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The heading content in content item.
+ *
+ * @element dds-cta-section-item-heading
+ */
+@customElement(`${ddsPrefix}-cta-section-item-heading`)
+class DDSCTASectionItemHeading extends StableSelectorMixin(DDSContentItemHeading) {
+  /**
+   * The shadow slot this logo UI should be in.
+   */
+  @property({ reflect: true })
+  slot = 'heading';
+
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'heading');
+    }
+    if (!this.hasAttribute('aria-level')) {
+      this.setAttribute('aria-level', '4');
+    }
+    super.connectedCallback();
+  }
+
+  render() {
+    return html`
+      <div class="${prefix}--content-item__heading">
+        <slot></slot>
+      </div>
```

Comment:
`<cta-section-item-heading>` itself is a DOM element and thus rendering `<div class="bx--content-item">` will be a duplicate. We can use a tag selector to style `<cta-section-item>`: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/docs/coding-conventions.md#custom-element-itself-as-an-eleement

```suggestion
      <slot></slot>
```